packages/desktop/views/dashboard/settings/views/help/Help.svelte

```diff
@@ -2,15 +2,25 @@
     import { HR } from 'shared/components'
     import { HelpAndInfoRoute } from '@core/router'
     import HelpSection from './HelpSection.svelte'
+    import Diagnostics from './Diagnostics.svelte'
+    import ErrorLog from './ErrorLog.svelte'
     import features from '@features/features'
     import { DISCORD_URL, DOCUMENTATION_URL, FAQ_URL, ISSUE_REPORT_URL } from '@contexts/settings'
 
     const settings: {
         component: unknown
         childRoute: HelpAndInfoRoute
-        actionLocale: string
-        url: string
+        actionLocale?: string
+        url?: string
     }[] = [
```

Comment:
Should use a defined interface as suggested above

Reply:
Can clean this up when we merge all the settings pages into a generic component.